# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/MacroAssembler.h
# Commit: 5737845e8af2
# Full Hash: 5737845e8af228fab6e5e622863663faa50f93e8
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2020-10-30 16:09:44
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Part 5: Remove remaining uses of Frame::tls. r=lth
#   
#   
#   Here we remove remaining uses of Frame::tls. There are many places where
#   we use it: in debug frames, in profiling frames, in jit activation, etc.
# ==============================================================================

diff -r 5dfb9fc15fba -r 5737845e8af2 js/src/jit/MacroAssembler.h
--- a/js/src/jit/MacroAssembler.h	Fri Oct 23 18:17:37 2020 +0000
+++ b/js/src/jit/MacroAssembler.h	Fri Oct 23 18:17:38 2020 +0000
@@ -4410,12 +4410,11 @@
   explicit WasmABIArgIter(const VecT& types)
       : ABIArgIterBase<VecT, WasmABIArgGenerator>(types) {}
 };
-
 }  // namespace jit
 
 namespace wasm {
-TlsData* ExtractCalleeTlsFromFrameWithTls(Frame* fp);
-TlsData* ExtractCallerTlsFromFrameWithTls(Frame* fp);
+const TlsData* ExtractCalleeTlsFromFrameWithTls(const Frame* fp);
+const TlsData* ExtractCallerTlsFromFrameWithTls(const Frame* fp);
 }  // namespace wasm
 
 }  // namespace js